WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -42,7 +42,7 @@
     <% end %>
   <% else %>
 
-    <% if @competition.registration_opened? %>
+    <% if @competition.registration_opened? && @competition.showAtAll? %>
```

Comment:
This removes the form, but does this actually prevent organizers from registering for a competition whose registration is open, but `showAtAll` is false?